docs/CLI.md

```diff
@@ -309,7 +312,13 @@ Allows the test suite to pass when no files are found.
 
 ### `--projects <path1> ... <pathN>`
 
-Run tests from one or more projects, found in the specified paths; also takes path globs. This option is the CLI equivalent of the [`projects`](configuration#projects-arraystring--projectconfig) configuration option. Note that if configuration files are found in the specified paths, _all_ projects specified within those configuration files will be run.
+Run tests from one or more projects, found in the specified paths; also takes path globs. This option is the CLI equivalent of the [`projects`](configuration#projects-arraystring--projectconfig) configuration option.
+
+:::note
+
+ Note that if configuration files are found in the specified paths, _all_ projects specified within those configuration files will be run.
```

Comment:
I think we can skip the "Note that" starts to all these sentences

Reply:
Done!

Reply:
These have been removed from all .md files that have Note admonitions.